.travis.yml

```diff
@@ -3,5 +3,12 @@ language: node_js
 node_js:
   - 6
 
+sudo: required
+services:
+  - docker
+
+before_script:
+  - docker build -t studio-frontend --no-cache .
```

Comment:
This builds the image before the script runs, so if we wanted to run tests on the image, we could do it in the script step, when the time comes.